easy_phi_client.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+import urllib
+import logging
+
+import settings
+
+
+class PythonClient(object):
+    baseURL = settings.base_url
+
+    def __init__(self, api_token):
+        self.logger = logging.getLogger(__name__)
+        self.api_token = api_token
+
+    def _request(self, api_func, params=None):
+
+        # build ULR of API request, which looks like:
+        # https://${base_url}/{api_func}?param1=foo&param2=bar
+        query = '' if params is None else '?' + urllib.urlencode(params)
+        url = ''.join((self.baseURL, api_func, query))
+        self.logger.debug("_build_url: {}:".format(url))
+
+        try:
+            response = urllib.urlopen(url)
+        except IOError:
+            self.logger.error('Failed to open url: {}'.format(url))
+            raise
+        response = response.read()
+        self.logger.debug("Response from: {}".format(response))
+
+        return response
+
+    def get_platform_info(self):
+        data = self._request('/api/v1/info')
+        return data
+
+    def get_module_info(self, slot):
+        data = self._request('/api/v1/module?slot={}'.format(slot))
```

Comment:
please use params instead of formatting URL
also, assert that slot is a positive small integer won't hurt